guillotina/content.py

```diff
@@ -101,17 +101,19 @@ def __init__(
         type_name="",
         schema=None,
         behaviors=None,
-        add_permission=DEFAULT_ADD_PERMISSION,
-        allowed_types=None,
+        add_permission: str = DEFAULT_ADD_PERMISSION,
+        allowed_types: Optional[List[str]] = None,
+        globally_disallowed: bool = False,
```

Comment:
Can we use positive naming maybe? Rename `globally_disallowed` to `globally_addable` and default to `True` then?